hooks/validator/pkg/validator/validator.go

```diff
@@ -0,0 +1,31 @@
+package validator
+
+import (
+	of "github.com/open-feature/golang-sdk/pkg/openfeature"
+)
+
+type validator interface {
+	IsValid(flagEvaluationDetails of.EvaluationDetails) error
+}
+
+// Hook validates the flag evaluation details After flag resolution
+type Hook struct {
+	Validator validator
+}
+
+func (h Hook) Before(hookContext of.HookContext, hookHints of.HookHints) (*of.EvaluationContext, error) {
```

Comment:
Is it required to add unused hooks?

Reply:
The methods will still need to be present to satisfy the interface even if they are unused

Reply:
ok, thanks.